Tools/TestWebKitAPI/Tests/WebKitCocoa/FindInPage.mm

```diff
@@ -913,4 +913,44 @@ static void swizzledPerformTextSearchWithQueryString(id, SEL, NSString *, UIText
     EXPECT_TRUE(findSessionSupportsReplacement());
 }
 
+static bool hasPerformedTextSearchWithQueryString = false;
+
+static void swizzledPerformTextSearchInPDFWithQueryString2(id, SEL, NSString *, UITextSearchOptions *, id<UITextSearchAggregator> aggregator)
```

Comment:
Can we just update the implementation of `swizzledPerformTextSearchInPDFWithQueryString` rather than creating a `2` that's almost the exact same?